### PR TITLE
Docs of new Java SDK features

### PR DESCRIPTION
**Pull request by novalisdenahi**
### Description

Java-SDK default user option docs update.

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/docs/sdk-reference/java.md [hunk 1]
```diff
@@ -58,28 +63,25 @@ The ConfigCat Java SDK is compatible with Java 8 and higher.
...
+| `mode(PollingMode)`                                           | Optional, sets the polling mode for the client. [More about polling modes](#polling-modes).                                                                                                                                                                                                        |
+| `logLevel(LogLevel)`                                          | Optional, defaults to `WARNING`. Sets the internal log level. [More about logging](#logging).                                                                                                                                                                                                      |
+| `flagOverrides(OverrideDataSourceBuilder, OverrideBehaviour)` | Optional, configures local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides).                                                                                                                                                                                 |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/java.md:77]:
```suggestion
| `flagOverrides(OverrideDataSourceBuilder, OverrideBehaviour)` | Optional, sets the local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides).                                                                                                                                                                                 |
```

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by erosb — COMMENTED** [review]
belepofázás done.

Further suggestion: you could use type inference for local variables (`var`), it makes the code samples.

Further-further suggestion: please include the minimum supported java version.

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

File: website/docs/sdk-reference/java.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ import com.configcat.*;
 ### 3. Create the _ConfigCat_ client with your _SDK Key_
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/java.md:39]: create and get ...

File: website/docs/sdk-reference/java.md [hunk 3]
```diff
@@ -60,6 +60,12 @@ You can safely shut down the client instance and release all associated resource
 ```java
 client.close()
 ```
+You can close all singleton client instances and release all associated resources on application exit.
+
+```java
+ConfigCatClient.closeAll();
+```
+This method has no effect on client's created with the deprecated constructor.
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/java.md:68]:
```suggestion
This method has no effect on the clients that were created with the deprecated constructor.
```

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

File: website/docs/sdk-reference/java.md (hunk 2 above)
Comment (sigewuzhere) [reply website/docs/sdk-reference/java.md]: Get the _ConfigCat_ client for your _SDK Key_

**Review by sigewuzhere — APPROVED** [review]

**Review by sigewuzhere — DISMISSED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Review by z4kn4fein — APPROVED** [review]

**Pull request merged**